packages/adp-tooling/src/writer/options.ts

```diff
@@ -14,9 +14,13 @@ import type {
     Content,
     CloudApp,
     ChangeInboundNavigation,
-    InternalInboundNavigation
+    InternalInboundNavigation,
+    CloudCustomTaskConfig,
+    CloudCustomTaskConfigTarget
 } from '../types';
 
+const vscodeUrl = 'https://REQUIRED_FOR_VSCODE.example';
```

Comment:
If this is a const, it should be uppercase letters, something like: `VSCODE_URL`. Otherwise, I don't see any reason for this extraction into a variable.

Reply:
Fixed in [216a14a](https://github.com/SAP/open-ux-tools/pull/2362/commits/216a14ab037ab88ab871d6d3620f7bfd88aa0e5f)